tests/fixtures/edx/server.py

```diff
@@ -0,0 +1,25 @@
+"""Server event factory definitions"""
+
+from factory import LazyAttribute, lazy_attribute
+
+from ralph.models.edx.server import ServerEventModel
+
+from .base import FAKE, BaseEventFactory
+
+
+class ServerEventFactory(BaseEventFactory):
+    """Factory for the ServerEventModel"""
+
+    class Meta:  # pylint: disable=missing-class-docstring
+        model = ServerEventModel
+
+    event_type = LazyAttribute(lambda self: self.context.path)
+
+    @lazy_attribute
+    def event(self):  # pylint: disable=no-self-use
+        """Return a randomly empty/filled POST/GET value"""
+
+        empty = '{"POST": {}, "GET": {}}'
+        post = '{"POST": {"input_e8bc718966e9441abc3ccf1a6429ee8b_2_1": ["choice_2"]}, "GET": {}}'
+        get = '{"POST": {}, "GET": {"rpp": ["50"], "page": ["1"]}}'
+        return FAKE.random_element([empty, post, get])
```

Comment:
Why not using a Param instead of a random choice?

Reply:
Good point, yes indeed.
This made me think that actually this part is not used in the tests. Maybe we could make it just one string ?